src/scss/_base.scss

```diff
@@ -1,33 +1,49 @@
+/// Fundamental styles for header
+@mixin oHeaderServicesBase() {
+	.o-header-services {
+		@include oTypographySans($scale: 0);
+		background-color: _oHeaderServicesGet('top-background');
+		-webkit-font-smoothing: antialiased; // sass-lint:disable-line no-vendor-prefixes
```

Comment:
I'm not sure why we don't include it in o-typography, to be honest. Maybe because it's [not widely compatible](https://caniuse.com/#search=font-smooth)? 
This was mostly copy pasta, so I don't have a good explanation.